drivers/spi/spi_smartbond.c

```diff
@@ -247,7 +287,7 @@ static const struct spi_driver_api spi_smartbond_driver_api = {
 	.release = spi_smartbond_release,
 };
 
-static int spi_smartbond_init(const struct device *dev)
+static int spi_smartbond_resume(const struct device *dev)
```

Comment:
Is`spi_context_cs_configure_all` required when `spi_smartbond_resume` is called? Shouldn't be called only once in `spi_smartbond_init`?